src/main/resources/com/github/mylibrelab/ui/icons/redo.svg

```diff
@@ -0,0 +1 @@
+<svg ti:v="1" width="16" height="16" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg" xmlns:ti="urn:schemas-jetbrains-com:tisvg"><rect id="frame" width="16" height="16" fill="none"/><g fill-rule="evenodd"><path d="M6.5 1c1.469 0 2.85.572 3.889 1.611l.611.612V2h1.414L15 4.586V10H9.586L7 7.414V6h1.121l-.561-.562a1.502 1.502 0 0 0-2.121 2.123L10 12.121v.93L8.051 15h-.828l-4.612-4.611A5.464 5.464 0 0 1 1 6.5C1 3.468 3.468 1 6.5 1z" fill="#F4F4F4"/><path d="M2 6.5a4.5 4.5 0 0 1 7.682-3.182L12 5.637V3l2 2v4h-4L8 7h2.535L8.268 4.732a2.5 2.5 0 1 0-3.536 3.536l4.319 4.318L7.637 14 3.318 9.682A4.49 4.49 0 0 1 2 6.5z" fill="#00529C"/></g></svg>
```

Comment:
These undo/redo icons look a bit out of place (Probably because they are from the NetIcons group. I would avoid icons which have a group name with a period in them. All newer icons have paths with /.